time/src/main/java/net/jqwik/time/api/Dates.java

```diff
@@ -81,4 +100,13 @@ public static MonthDayArbitrary monthDays() {
 		return new DefaultMonthDayArbitrary();
 	}
 
+	/**
+	 * Create an arbitrary that generates instances of {@linkplain java.time.Period}.
```

Comment:
Maybe you can add a short note about the possible ranges of periods being generated.